app/code/Magento/InventorySalesApi/Api/Data/SalesEventInterface.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventorySalesApi\Api\Data;
+
+/**
+ * Represents the sales event that brings to appending reservations.
+ *
+ * @api
+ */
+interface SalesEventInterface
+{
+    const TYPE_QUOTE = 'quote';
```

Comment:
in the very first draft of implementation I had both **event type** and **event object type** but then we decided to remove event object type; maybe we can consider re-introducing the idea of the event object type to make it more clear what the object id refers to;

since events are usually named in the past tense I would suggest using `order_placed` for event type; here is my proposal:

```
interface SalesEventInterface
{
    /**#@+
     * Constants for event types
     */
    const TYPE_ORDER_PLACED = 'quote';
    /**#@-*/

    /**#@+
     * Constants for event object types
     */
    const OBJECT_TYPE_QUOTE = 'quote';
    /**#@-*/

    public function getType(): string;

    public function getObjectType(): string;

    public function getObjectId(): string;
}
```

In the `metadata` of a reservation we would have values like:

```
order_placed:quote:10
```

What do you think @maghamed ?

Reply:
approved